src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -156,7 +156,6 @@ public partial class SendPacketsElement
     public partial class Socket : System.IDisposable
     {
         public Socket(System.Net.Sockets.AddressFamily addressFamily, System.Net.Sockets.SocketType socketType, System.Net.Sockets.ProtocolType protocolType) { }
-        public Socket(System.Net.Sockets.SocketInformation socketInformation) { }
```

Comment:
Whether or not there is a blocking Win32 call here, this API is a Win32-ism and should be removed (along with `Socket.DuplicateAndClose`, `class SocketInformation`, and `enum SocketInformationOptions`).